src/main/java/no/nav/syfo/rest/ressurser/FastlegeAzureAD2Ressurs.java

```diff
@@ -44,22 +46,48 @@ public FastlegeAzureAD2Ressurs(
     }
 
     @GetMapping(produces = APPLICATION_JSON_VALUE)
-    public Fastlege finnFastlegeAazure(@RequestParam(value = "fnr") String fnr) {
+    public Fastlege finnFastlegeAazure(
+            @RequestHeader MultiValueMap<String, String> headers,
+            @RequestParam(value = "fnr", required = false) String fnr
+    ) {
         metrikk.tellHendelse("finn_fastlege");
 
-        kastExceptionHvisIkkeTilgang(fnr);
+        String requestedPersonIdent;
```

Comment:
Kan evt refaktorere ut i metode som returnerer requestedPersonIdent eller kaster IllegalArgumentException som kan brukes begge steder her

Reply:
Laget egen metode for det nå.